plugins/rest-plugin/src/main/java/ru/sbtqa/tag/api/EndpointEntryReflection.java

```diff
@@ -0,0 +1,161 @@
+package ru.sbtqa.tag.api;
+
+import java.io.IOException;
+import java.lang.annotation.Annotation;
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.aeonbits.owner.ConfigFactory;
+import org.apache.commons.io.IOUtils;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import ru.sbtqa.tag.api.annotation.Body;
+import ru.sbtqa.tag.api.annotation.FromResponse;
+import ru.sbtqa.tag.api.annotation.Header;
+import ru.sbtqa.tag.api.annotation.ParameterType;
+import ru.sbtqa.tag.api.annotation.Query;
+import ru.sbtqa.tag.api.annotation.Stashed;
+import ru.sbtqa.tag.api.annotation.Validation;
+import ru.sbtqa.tag.api.annotation.applicators.Applicator;
+import ru.sbtqa.tag.api.annotation.applicators.ApplicatorHandler;
+import ru.sbtqa.tag.api.annotation.applicators.FromResponseApplicator;
+import ru.sbtqa.tag.api.annotation.applicators.StashedApplicator;
+import ru.sbtqa.tag.api.exception.RestPluginException;
+import ru.sbtqa.tag.api.properties.ApiConfiguration;
+import static ru.sbtqa.tag.api.utils.ReflectionUtils.get;
+import static ru.sbtqa.tag.api.utils.ReflectionUtils.set;
+import ru.sbtqa.tag.qautils.reflect.FieldUtilsExt;
+
+public class EndpointEntryReflection {
+
+    private static final Logger LOG = LoggerFactory.getLogger(EndpointEntryReflection.class);
+    private static final ApiConfiguration PROPERTIES = ConfigFactory.create(ApiConfiguration.class);
+    private static final String BOM = "\uFEFF";
+
+    private EndpointEntry endpoint;
+    private List<Field> fields;
+
+    public EndpointEntryReflection(EndpointEntry endpoint) {
+        this.endpoint = endpoint;
+        this.fields = FieldUtilsExt.getDeclaredFieldsWithInheritance(endpoint.getClass());
+    }
+
+    public void applyAnnotations() {
+        for (Field field : fields) {
+            field.setAccessible(true);
+
+            ApplicatorHandler<Applicator> applicators = new ApplicatorHandler<>();
+            for (Annotation annotation : field.getAnnotations()) {
+                if (annotation instanceof FromResponse) {
+                    applicators.add(new FromResponseApplicator(endpoint, field));
+                } else if (annotation instanceof Stashed) {
+                    applicators.add(new StashedApplicator(endpoint, field));
+                } else {
+                    continue;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this redundant jump. ([squid:S3626](https://sonarcloud.io/organizations/sbtqasonarkey/rules#rule_key=squid%3AS3626))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=ru.sbtqa.tag%3Apage-factory-2&pullRequest=81&issues=AWU2KVJuW2AOP0CkD5Fw&open=AWU2KVJuW2AOP0CkD5Fw)

[ISSUE_KEY]: AWU2KVJuW2AOP0CkD5Fw